src/main/pages/che-7/administration-guide/assembly_monitoring-che.adoc

```diff
@@ -15,41 +15,9 @@ summary:
 
 :context: monitoring-che
 
-{prod-short} can expose certain data as metrics, that can be processed by Prometheus
```

Comment:
Can you explain the motivation of removing this information from assembly_monitoring-che.adoc

Reply:
Good point. This chapter should also be about Enabling the production of metrics in Che, indeed. I will make it more clear then.

Reply:
handled feedback from @skabashnyuk - add visibility to enabling and exposing che metrics in last commit